README.md

```diff
@@ -0,0 +1,70 @@
+<div align="center">
+  <img src="https://raw.githubusercontent.com/Castro-Fidel/PortWINE/master/data_from_portwine/img/gui/portproton.svg" width="64">
+  <h1 align="center">PortProton on Flatpak</h1>
+  <p align="center">Project designed to make it easy and convenient to run Windows games on Linux for both beginners and advanced users.
+The project strives to make launching games (and other software) as simple as possible, but at the same time provides flexible settings for advanced users.</p>
+</div>
+
+## Installation
+
+   ```sh
+   flatpak install flathub ru.linux_gaming.PortProton
+   ```
+
+## Running
+Launch PortProton from your desktop menu, or via command line:
+```
+flatpak run ru.linux_gaming.PortProton
+```
+
+## Building
+
+To compile PortProton as a Flatpak, you'll need both [Flatpak](https://flatpak.org/) and [Flatpak Builder](http://docs.flatpak.org/en/latest/flatpak-builder.html) installed. Once you manage that, do the following...
+
+0. Clone this repository and `cd` into it
+1. Add the git submodules
+   ```sh
+   git submodule init
+   git submodule update
+   ```
+2. Compile the flatpak
+   ```sh
+   flatpak-builder --repo=portproton --force-clean --install-deps-from=flathub build-dir ru.linux_gaming.PortProton.yml
+   ```
+3. Add the local repo and install the flatpak
+   ```sh
+   flatpak remote-add portproton portproton --no-gpg-verify
+   flatpak install portproton ru.linux_gaming.PortProton
+   ```
+
+### MangoHud
+
+To enable MangoHud user conf support do this command
+
+```
+flatpak override --user --filesystem=.config/MangoHud:ro ru.linux_gaming.PortProton
+```
+
+## Wine sources used in PortProton:
+
+* https://github.com/Kron4ek/Wine-Builds
+* https://dl.winehq.org/wine/source
+* https://github.com/wine-staging/wine-staging
+* https://github.com/Frogging-Family/wine-tkg-git
+* https://github.com/Kron4ek/wine-tkg
+* https://github.com/ValveSoftware/wine
+* https://github.com/ValveSoftware/Proton
+* https://github.com/GloriousEggroll/proton-ge-custom
+* https://github.com/varmd/wine-wayland
+* https://github.com/Kron4ek/wine-wayland
+* https://gitlab.collabora.com/alf/wine/-/tree/wayland
+
+
+## Building wine from source code
+
+All sources as well as tools for building custom versions of Wine (Wine LG) and Proton (Proton LG) used in PortProton can be found in the [wine buillds](https://github.com/Castro-Fidel/wine_builds) repository.
+
+## What tarball downloaded in startup ?
+
+During the first run of PortProton the file PortWINE-master.tar.gz is downloaded, these are just scripts that are downloaded from gitlab or github, more details in [upstream](https://github.com/Castro-Fidel/PortProton_ALT/blob/main/portproton#L168).
```

Comment:
I saw a script from here downloading another tarball containing the actual wine proton etc. libraries.

Can you please add, how that one is produced as well?

Reply:
This script downloads the github master branch (gitlab is a priority because most PortProton users live in the CIS where there are problems with github), and as I wrote to you in the future you can download the master branch yourself and PortProton will just unpack it.

Reply:
How to build custom versions of Wine and Proton used in PortProton I also described above in the Readme

Reply:
I don't see which script to run and how to produce the tarballs here? https://github.com/Castro-Fidel/wine_builds/releases


What should I run if I want to produce my own https://github.com/Castro-Fidel/wine_builds/releases/download/PROTON_LG_9-4/PROTON_LG_9-4.tar.xz ?

Reply:
In releases there are already ready assemblies which are downloaded by PortProton build scripts in the repository first it needs to be cloned and then built

Reply:
> repository first it needs to be cloned and then built

Yes please document what script to run and how. The readme there has a bunch of links and no other information.

Reply:
So I ran the scripts from the repo the first script needs debootstrap and perl to run and the third one needs autoconf to run. These needs to be documented.

After running them I managed to produce a copy of `WINE_LG_9-7.tar.xz` but this one came as an empty archive, why?

![image](https://github.com/flathub/ru.linux_gaming.PortProton/assets/62639087/45e52e27-7705-4fbe-9815-52487345e5ec)

Reply:
The last script seems to have some kind of dependency, does it expect a folder?

```
mv: cannot stat 'GE-Proton9.4/files': No such file or directory
mv: cannot stat 'GE-Proton9.4/PATENTS.AV1': No such file or directory
rm: cannot remove 'GE-Proton9.4': No such file or directory
./04_ge_to_lg.sh: line 36: PROTON_LG_9.4/version: No such file or directory
rm: cannot remove 'PROTON_LG_9.4/share/default_pfx': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/*/cmake': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/*/fst': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/*/glslang': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/*/graphene-1.0': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/*/pkgconfig': No such file or directory
cat: PROTON_LG_9.4/lib/wine/vkd3d-proton/version: No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib/vkd3d/*': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/lib/vkd3d/': No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib/wine/vkd3d-proton/*': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/lib/wine/vkd3d-proton/': No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib64/vkd3d/*': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/lib64/vkd3d/': No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib64/wine/vkd3d-proton/*': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/lib64/wine/vkd3d-proton/': No such file or directory

Creating and compressing VKD3D archive...
xz: Reduced the number of threads from 12 to 1 to not exceed the memory usage limit of 1,475 MiB
cat: PROTON_LG_9.4/lib/wine/dxvk/version: No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib/wine/nvapi/version': No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib/wine/nvapi/*': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/lib/wine/nvapi/': No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib/wine/dxvk/*': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/lib/wine/dxvk/': No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib64/wine/nvapi/version': No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib64/wine/nvapi/*': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/lib64/wine/nvapi/': No such file or directory
mv: cannot stat 'PROTON_LG_9.4/lib64/wine/dxvk/*': No such file or directory
rm: cannot remove 'PROTON_LG_9.4/lib64/wine/dxvk/': No such file or directory

Creating and compressing DXVK archive...
xz: Reduced the number of threads from 12 to 1 to not exceed the memory usage limit of 1,475 MiB

Creating and compressing archives...
tar: PROTON_LG_9.4: Cannot stat: No such file or directory
xz: Reduced the number of threads from 12 to 1 to not exceed the memory usage limit of 1,475 MiB
tar: Exiting with failure status due to previous errors
Done.
```

Reply:
And I couldn't figure out how these two are getting produced 

https://github.com/Castro-Fidel/wine_builds/releases/tag/plugins_v15 

https://github.com/Castro-Fidel/wine_builds/releases/tag/PROTON_STEAM_8.0-5

Reply:
Ok fine then once you install the license files per the other comment.

Reply:
> Ok fine then once you install the license files per the other comment.

Proton GE license added to the new Proton LG version, as well as a fixed build

https://github.com/Castro-Fidel/wine_builds/releases/tag/PROTON_LG_9-5-1

Reply:
Looks good thanks.